### PR TITLE
Expanded directory structure of get method,  and added verbs and nouns of Gurmukhi and Shahmukhi (Punjabi)

### PR DESCRIPTION
**Pull request by SethiShreya**
Signed off by Shreya Sethi
shreyasethi47@gmail.com

- Bifurcated Punjabi language to Gurmukhi and Shahmukhi
- Added noun and verb query to Shahmukhi and verb query to Gurmukhi
- Expanded the get command to visit these bifurcated queries 
- Also added a check for format.py, run formatting only when format.py is present
- Tested the get command on my end

Test it with query
```scribe-data get --lang gurmukhi -dt verbs```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,4 @@ package-lock.json
 # MARK: Test Files
 
 tests_output
+scribe_data_json_export/*
```
Comment (andrewtavis) [inline .gitignore:39]: Great idea, @SethiShreya :) I'll also remove the current ones from the repo :)

Comment (andrewtavis) [reply .gitignore:39]: Or hmmm, not quite yet as we need the current translations. But soon :)

Comment (andrewtavis) [reply .gitignore:39]: Removed all the data with Wikidata as its source and also the SQLite directory :) Also added all other output types.

File: src/scribe_data/language_data_extraction/Punjabi/Shahmukhi/nouns/query_nouns.sparql [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  # MARK: Gender(s)
+
+  OPTIONAL {
+    ?lexeme wdt:P5185 ?nounGender .
+    FILTER NOT EXISTS {
+      ?lexeme wdt:P31 wd:Q202444 .
+    }
+  } .
+
+  FILTER(lang(?singular) = "pnb")
```
Comment (andrewtavis) [inline src/scribe_data/language_data_extraction/Punjabi/Shahmukhi/nouns/query_nouns.sparql:38]: Doing a quick fix for this, but these filters need to be close to their selections and if there's an OPTIONAL block then they need to be inside. If they're outside the OPTIONAL, then only nouns that have plurals will be returned as those that don't will be filtered out.

File: src/scribe_data/wikidata/query_data.py [hunk 3]
```diff
@@ -234,11 +234,10 @@ def query_data(
 
                 results_final.append(r_dict)
 
+            path = os.path.dirname(q)
```
Comment (andrewtavis) [inline src/scribe_data/wikidata/query_data.py:237]: I'm going to revert these changes for now, @SethiShreya, but will reference them in the future! Ideally we would have formatting files for all of the languages anyway, and this is coming once I'm done reworking the formatting process :)

Thanks for thinking of this, and again I'll reference this later 😊

**Review by andrewtavis — APPROVED** [review]
Great thinking here, @SethiShreya! Again sorry that the query data edits are reverted as I understand that you want this to start working, but we'll get this done by the end of next week 😊

File: src/scribe_data/wikidata/query_data.py (hunk 3 above)
Comment (SethiShreya) [reply src/scribe_data/wikidata/query_data.py]: Checking the formatting file was just a small part, the main task these PR was doing is to run those bifurcated queries like Gurmukhi and Shahmukhi in Punjabi, did you reviewed it, and if it is not important right now then we will surely work on that later, that's not an issue :)

**Pull request merged**